src/lib/draw/point/point_tool.ts

```diff
@@ -81,17 +82,22 @@ export class PointTool {
   // Note there's no way to "edit an existing point." Just call this for a new
   // or existing point; the state just depends on the cursor anyway.
   start() {
-    this.active = true;
+    this.setActivity(true);
     // TODO Figure out where the cursor is and immediately draw? To be useful,
     // the user has to move their mouse anyway; it doesn't matter much
   }
 
   stop() {
     this.cursor = null;
-    this.active = false;
+    this.setActivity(false);
```

Comment:
I think this all should be fine, and the tests will tell us if not. If for some reason we called `stop()` on a tool when it wasn't active in the first place, this will mess up the store. But we shouldn't do that right now, so it's OK